test/integration/images/sti-fake-onbuild/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM sti_test/sti-fake
+
+ONBUILD RUN touch /sti-fake/src/onbuild
+
+RUN touch /sti-fake/src/index.html
```

Comment:
@soltysh if you look at the pastebin you can see that it is able to copy the sources specified in the LABEL but it never moves them to /tmp like it does for the top test (sti-fake).  I also notice that when it `tar`s up the source it does so without them being nested under a src directory.  (Compare lines 27 and 69)